test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -22,30 +23,103 @@
 # E0326 11:41:27.125006742  636480 fork_posix.cc:70]           Fork support is only compatible with the epoll1 and poll polling strategies
 # Maybe: https://github.com/grpc/grpc/issues/29044
 
+eth_account_number = 2
+sif_account_number = 1
+transaction_number = 2
+
+# build the transfer matrix 
```

Comment:
To run load test, you may set these number as different value. For example, 1000 ethereum account. It is hard to write such matrix manually.

Reply:
You can use list comprehensions: `[[2 for y in range(1000)] for x in range(1000)]`.

Reply:
I did this in the python REPL:

```python
[[2 for _ in range(3)] for _ in range(10)]
```

That's cool because if you don't need the value, you can just replace it with an underscore. It makes code easier to understand.